src/content/en/fundamentals/performance/rail.md

```diff
@@ -36,11 +37,15 @@ Understand how users perceive performance delays:
   <tbody>
     <tr>
       <td data-th="Delay">0 - 16ms</td>
-      <td data-th="User Reaction">Given a screen that is updating 60 times per second, this window represents the time to get a single frame to the screen (Professor Math says "1000/60 ~= 16"). People are exceptionally good at tracking motion, and they dislike it when the expectation of motion isn’t met, either through variable frame rates or periodic halting.</td>
+      <td data-th="User Reaction">People are exceptionally good at tracking
+      motion, and they dislike it when animations aren't smooth. Users
+      perceive animations as smooth so long as 60 new frames are rendered
+      every second. That's 16ms per frame, including the time it takes for
+      the browser to paint the new frame to the screen.</td>
```

Comment:
Might be worth adding something along the lines of:

"including the time it takes for the browser to paint the new frame to the screen, leaving a web app about 10ms to produce a frame"